docs/semgrep-cloud-platform/scm.md

```diff
@@ -83,7 +83,14 @@ PR or MR comments are comments or suggestions made by Semgrep Cloud Platform in
 
 To enable this feature within self-hosted SCMs behind firewalls or VPNs (Virtual Private Networks), follow these steps:
 
- 1. Add the IP address `52.34.175.113` to your VPN's allowlist. This IP address is **static and outbound**.
+ 1. Add the following IP addresses to your VPN's allowlist. These IP addresses are **static and outbound**:

```

Comment:
cc @zyannes I think this "outbound" is true. Can you confirm?

Reply:
Technically the `52.34.175.113` IP is strictly outbound, while the others are outbound AND inbound. 

From the customer's perspective, the IPs need to be added to their ingress allowlist in order for SCM comments to be posted on PRs/MRs. 

If the customers have an egress allowlist (e.g. for blocking outgoing connections in their CI workers) then the other 4 IPs can be added so that their CI workers can communicate with the Scan API to fetch the scan rules, upload findings, etc.

Reply:
Thanks for that bit. Here's what the steps look like now:
<img width="654" alt="Snag_e09fd8" src="https://github.com/returntocorp/semgrep-docs/assets/20766840/5eb5f770-abb7-4c28-9cad-c1c8c2530c26">

Reply:
Looks great! Thanks for updating it!